hub/api/dataset_utils.py

```diff
@@ -0,0 +1,39 @@
+from typing import Union
+from hub.core import compression
+from hub.util.exceptions import InvalidCompressor
+
+
+def get_compressor(
+    compressor_name: str, **kwargs
+) -> Union[compression.BaseImgCodec, compression.BaseNumCodec]:
+    f"""Get compressor object
+
+    Example:
+        compressor = get_compressor('lz4', acceleration=2)
+
+    Args:
+        compressor_name (str): lowercase name of the required compressor
```

Comment:
why lowercase? also, you should be consistent with capitalization and punctuation.